R/Module-Characterization.R

```diff
@@ -144,7 +141,6 @@ CharacterizationModule <- R6::R6Class(
                                             endAnchor = c("cohort end", "cohort end")
                                           ),
                                           minPriorObservation = 0,
-                                          minCharacterizationMean = 0,
```

Comment:
We'll still need to account for the `minCharacterizationMean` parameter for the module since it is used here: https://github.com/OHDSI/Characterization/blob/v2.0.0/R/RunCharacterization.R#L150.